apps/portal/src/content/docs/components/file-additions-trigger.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: File Additions Trigger
+description: File Additions Trigger component documentation
+---
+
+The File Additions Trigger component provides a UI element for creating new files or uploading files. It uses a dropdown menu to offer these options.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<FileAdditionsTrigger
+  useTranslationStore={() => ({ t: (key, fallback) => fallback })}
+  pathNewFile="/new-file"
+  pathUploadFiles="/upload-files"
```

Comment:
The formatting of the `ComponentExample` code highlighting is a bit off an expects 4 spaces instead of 2. 2 is fine everywhere else
```suggestion
    useTranslationStore={() => ({ t: (key, fallback) => fallback })}
    pathNewFile="/new-file"
    pathUploadFiles="/upload-files"
```